apis/io-engine/protobuf/v1/stats.proto

```diff
@@ -29,6 +31,17 @@ message NexusIoStatsResponse {
   repeated IoStats stats = 1;
 }
 
+/// Replica IO stats response.
+message ReplicaIoStatsResponse {
+  repeated ReplicaIoStats stats = 1;
+}
+
+// Replica IO stats struct.
+message ReplicaIoStats {
+  optional string entity_id = 1; // Entity ID of the replica, If any.
```

Comment:
If available, what is entity id w.r.t replica? The owner Volume?

Reply:
Yes @dsharma-dc